app/modules/node.c

```diff
@@ -295,6 +307,7 @@ const LUA_REG_TYPE node_map[] =
   { LSTRKEY( "input" ), LFUNCVAL( node_input ) },
   { LSTRKEY( "output" ), LFUNCVAL( node_output ) },
   { LSTRKEY( "readvdd33" ), LFUNCVAL( node_readvdd33) },
+  { LSTRKEY( "dsleepsetoption" ), LFUNCVAL( node_deepsleep_setoption) },
```

Comment:
Maybe can use dsleep(us, option) one time sovle every thing. Default option is 0. It is more easy and save ram.